src/java/org/apache/cassandra/utils/ObjectSizes.java

```diff
@@ -24,23 +24,25 @@
 import java.net.UnknownHostException;
 import java.nio.ByteBuffer;
 
-import org.github.jamm.MemoryLayoutSpecification;
 import org.github.jamm.MemoryMeter;
+import org.github.jamm.MemoryMeter.ByteBufferMode;
+import org.github.jamm.MemoryMeter.Guess;
+
+import static org.github.jamm.MemoryMeterStrategy.MEMORY_LAYOUT;
+import static org.github.jamm.utils.ArrayMeasurementUtils.computeArraySize;
 
 /**
  * A convenience class for wrapping access to MemoryMeter
  */
 public class ObjectSizes
```

Comment:
So after seeing the InMemoryTrie not using the `ObjectSizes` wrapper I have a question... should we always use the wrapper and make it clear in the javadoc here?